### PR TITLE
[1.13] Tune fluent bit's configuration

### PR DESCRIPTION
**Pull request by philipnrmn**
## High-level description

Backport of #5131

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS-51758](https://jira.mesosphere.com/browse/DCOS-51758) Masters and many agents got unreachable after upgrading soak113s from 1.13.0-alpha to 1.13.0-beta

## Related tickets (optional)

Other tickets related to this change:

  - [DCOS_OSS-<number>](https://jira.mesosphere.com/browse/DCOS_OSS-<number>) Foo the Bar so it stops Bazzing.

## Checklist for all PRs

  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: this is a configuration change, no dc/os code is involved here. 
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change: this simply ensures that fluentbit will not cause performance issues on startup, there is no user-facing change
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

### CODE REVIEW

File: gen/dcos-config.yaml [hunk 1]
```diff
@@ -2785,6 +2785,7 @@ package:
           # are further forwarded into elasticsearch, where fields with leading
           # underscore are reserved for internal use.
           Strip_Underscores On
+          Read_From_Tail    On
```
Comment (jdef) [inline gen/dcos-config.yaml]: Why this change instead of setting `DB` to some value in order to configure a persistent cursor?

Comment (philipnrmn) [reply gen/dcos-config.yaml]: We should enable the DB (this will require setting some build flags, as [FLB_SQLDB is not enabled by default](https://github.com/fluent/fluent-bit/blob/master/CMakeLists.txt#L62)), but we would still need the `Read_From_Tail` flag - when a user upgrades to 1.13, they will have logs on disk which are unread, and it's this read that caused issues in soak.

Comment (jdef) [reply gen/dcos-config.yaml]: Understood. It sounds like we should have a JIRA (if we don't already) to track the tasks associated with enabling the `DB` flag here. Can you link such to this comment thread?

**Comment by philipnrmn** [comment]
@jdef, the reason this isn't marked ready for review is that those things are not done.

File: gen/dcos-config.yaml (hunk 1 above)
Comment (philipnrmn) [reply gen/dcos-config.yaml]: https://jira.mesosphere.com/browse/DCOS-51855

**Comment by philipnrmn** [comment]
@mesosphere-mergebot label ready for review

**Comment by philipnrmn** [comment]
@gracedo and @branden, would you kindly review this PR?
